content/applications/inventory_and_mrp/inventory/shipping_receiving/reservation_methods/before_scheduled_date.rst

```diff
@@ -0,0 +1,153 @@
+=================================
+Before scheduled date reservation
+=================================
+
+.. _inventory/reservation_methods/before-scheduled-date:
+
+.. |SO| replace:: :abbr:`SO (Sales Order)`
+
+The *Before scheduled date* reservation method allows users to select a specific number of days that
+act as the maximum number of days **before** a scheduled delivery date, when products included in a
+sales order (SO) should be reserved.
+
+.. seealso::
+   :doc:`About reservation methods <../reservation_methods>`
+
+Configuration
+=============
+
+To set the reservation method to *Before scheduled date*, navigate to :menuselection:`Inventory app
+--> Configuration --> Operations Types`. Then, select the desired :guilabel:`Operation Type` to
+configure, or create a new one by clicking :guilabel:`New`.
+
+In the :guilabel:`General` tab, locate the :guilabel:`Reservation Method` field, and select
+:guilabel:`Before scheduled date`.
+
+.. image:: before_scheduled_date/before-scheduled-date-configuration.png
+   :align: center
+   :alt: Reservation method field on delivery order operation type form.
+
+.. note::
+   When the :guilabel:`Type of Operation` is changed to *Receipt* on an :guilabel:`Operations Type`
+   form, reservation methods are **not** available.
+
+Once selected, a new :guilabel:`Reserve before scheduled date` field appears below. From this field,
+the number of :guilabel:`days before` and :guilabel:`days before when starred` can be changed from
+the default `0`.
+
+Changing the :guilabel:`days before` value changes the maximum number of days before a scheduled
+date that products should be reserved.
+
+Changing the :guilabel:`days before when starred` value changes the maximum number of days before
+a scheduled date that products should be reserved for starred (favorited) transfers.
+
+.. example::
+   Here, the :guilabel:`days before` value is set to `2` days before, and the :guilabel:`days before
+   when starred` value is set to `3`.
```

Comment:
```suggestion
   If the :guilabel:`days before` value is set to `2` days before, and the :guilabel:`days before
   when starred` value is set to `3`.
```
I know you implemented my previous suggestion, but I think this current one is a bit better -- your call.

Reply:
Suggestion implemented ✅